sentry.properties

```diff
@@ -0,0 +1,3 @@
+defaults.project=wordpress-android
+defaults.org=a8c
+auth.token=2b46ef24f3cb448c92b383e0991b5787ab384536303b4e0eab618b3c2d735570
```

Comment:
Should this file be publically available here? Can we do something similar to `gradle.properties` here (and revoke this token)?

Reply:
Yep! I was thinking exactly the same.

Reply:
Yep for sure – this was supposed to be `.gitignore`'d. I got a warning about it right after but figured I'd leave it for testing since it was already there 🙃 

Will move over to mobile secrets.